src/event/event_kevent.c

```diff
@@ -730,6 +745,10 @@ _dispatch_kq_poll(dispatch_wlh_t wlh, dispatch_kevent_t ke, int n,
 		(void)avail;
 		const struct timespec timeout_immediately = {}, *timeout = NULL;
 		if (flags & KEVENT_FLAG_IMMEDIATE) timeout = &timeout_immediately;
+#ifdef EVFILT_USER
```

Comment:
This is confusing me.  If `EVFILT_USER` is present, the ersatz user filter shouldn't be used right?  Should this be `#ifndef EVFILT_USER`?

Reply:
You're right. I think perhaps I added something like this while debugging, didn't realize it was basically inoperative while testing, and didn't remove it. I've verified things work without this particular fragment.